dali/operators/image/crop/crop_mirror_normalize.h

```diff
@@ -110,10 +110,27 @@ class CropMirrorNormalize : public Operator<Backend>, protected CropAttr {
       inv_std_vec_ = { spec.GetArgument<float>("std") };
     }
 
+    DALI_ENFORCE(!mean_vec_.empty() && !inv_std_vec_.empty(),
+      "mean and standard deviation can't be empty");
+
     // Inverse the std-deviation
     for (auto &element : inv_std_vec_) {
       element = 1.f / element;
     }
+
+    // Handle irregular mean/std argument lengths
+    auto args_size = std::max(mean_vec_.size(), inv_std_vec_.size());
+    if (args_size > 0 && mean_vec_.size() != inv_std_vec_.size()) {
+      if (mean_vec_.size() == 1)
+        mean_vec_.resize(args_size, mean_vec_[0]);
+
+      if (inv_std_vec_.size() == 1)
+        inv_std_vec_.resize(args_size, inv_std_vec_[0]);
+
+      DALI_ENFORCE(mean_vec_.size() == args_size && inv_std_vec_.size() == args_size,
```

Comment:
I find the logic leading to this DALI_ENFORCE quite convoluted - I'd prefer to check the condition which you mention in the warning, not a result of some manipulation (see above).